src/main/java/com/bravo/user/service/PaymentService.java

```diff
@@ -0,0 +1,42 @@
+package com.bravo.user.service;
+
+import com.bravo.user.dao.model.Payment;
+import com.bravo.user.dao.model.mapper.ResourceMapper;
+import com.bravo.user.dao.repository.PaymentRepository;
+import com.bravo.user.model.dto.PaymentDto;
+import com.bravo.user.utility.PageUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Service;
+
+import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+
+@Service
+public class PaymentService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(UserService.class);
+
+    private final PaymentRepository paymentRepository;
+    private final ResourceMapper resourceMapper;
+
+    public PaymentService(PaymentRepository paymentRepository, ResourceMapper resourceMapper) {
+        this.paymentRepository = paymentRepository;
+        this.resourceMapper = resourceMapper;
+    }
+
+    public List<PaymentDto> retrieveByUserId(
+            final String userId,
+            final PageRequest pageRequest,
+            final HttpServletResponse httpResponse
+    ){
+        Page<Payment> paymentPage = paymentRepository.findAllByUserId(userId, pageRequest);
```

Comment:
- `paymentPage` could be `final`. 
- Variable names can be a bit more generic if you consider the context of the service they are used within. e.g. `paymentPage` can become `page`, `response`, etc.